theatre/studio/src/utils/stringify.ts

```diff
@@ -0,0 +1,23 @@
+import {tightJsonStringify} from './tightJsonStringify'
+
+export function stringify(input: any, indentJSON: boolean = true): string {
+  try {
+    return typeof input === 'string'
+      ? input
+      : typeof input === 'function'
+      ? input.toString()
+      : indentJSON
+      ? tightJsonStringify(input)
+      : JSON.stringify(input)
```

Comment:
I find these ternary chains hard to read because I'm not used to them and because of the lack of difference in indentation between the conditions and results. I would find this much easier to read if this was broken into nested ifs with named conditions or if else chains or something.

Reply:
We can add comments for helping the distinguishing between what is a boolean check versus value.

How long did you look at this? Was it confusing after a few seconds, or really hard to reason about due to not seeing this pattern much before?

e.g. is this a Nit or a Suggestion?

Reply:
I will say this is a nit. I still prefer if elses becaue I think it is more accessible for people, but I realized this is actually not that bad to read.